### ISSUE TITLE
Test prediction is constant

### ISSUE THREAD

**Issue by kei6** [MaybeShewill-CV/CRNN_Tensorflow#189]
After train the model for 50 epochs (each epoch is the whole dataset I mean), I use my test sample to predict, but the model always returns the same prediction for all of my testing set, even if I predict with my training set, it always return the same.
My training set is 1000 sample, the learning rate is 0.001.

**Comment by MaybeShewill-CV** [comment]
@kei6 You may update the repo and test again:)

**Comment by IngleJaya95** [comment]
@MaybeShewill-CV @kei6 Same thing happened in my case, I trained it for 3000 epochs and loss converges, but on testing, it always gives the same predictions. 
(Training set = 10 images)
Also on the same training images, when I test I get wrong and same predictions for all images.

**Comment by MaybeShewill-CV** [comment]
@IngleJaya95 You may pasted more details here:)

**Comment by IngleJaya95** [comment]
@MaybeShewill-CV I tried this model on Hindi language dataset, prepared dataset of 10 images as instructed and then trained the model for 3000 epochs with batch size 5 and lr=0.01, num_classes = 18 (as the count in char_dict.json )+1 = 19.
Model converges on loss 0.001114, I tested on one of the training images, just to see whether the model has learned or not.
![Hindi_1_0](https://user-images.githubusercontent.com/37269478/54987090-ec273200-4fd9-11e9-9a08-f9b760ddfe6e.jpg)
and I get results as
" I0326 15:10:15.245769 26679 test_shadownet.py:131] Predict image Hindi_1_2.jpg result ाइ"

I tried it on other 9 images also, and get the same predictions.

**Comment by MaybeShewill-CV** [comment]
@IngleJaya95 Have you updated the latest code and generated the char dict json file:)

**Comment by IngleJaya95** [comment]
@MaybeShewill-CV  yes, I have done that.

**Comment by MaybeShewill-CV** [comment]
@IngleJaya95 Then you may check if your tfrecords is correct and the char dict is correct. And your dataset is too small:)

**Comment by IngleJaya95** [comment]
@MaybeShewill-CV I checked both. Also, I tried it on training on synth90k, but the same scenario was observed there at the time of validation. Same prediction for all images.
